themes/ranked-choice-voting-for-texas-5dabb2094445ea66232bca7e/layout.html

```diff
@@ -179,7 +179,9 @@ <h3 class="home-hero-heading">Help bring ranked choice voting to Texas!</h3>
                 <h3>News</h3>
                 <ul class="top-news-list">
                   <li>We're actively recruiting &mdash; <a href="/volunteer_1">volunteer today!</a></li>
-                  <li>Next event: Tuesday, Dec. 7. <a href="/calendar_2">View our events calendar.</a></li>
+                  {% for event in site.upcoming_event_pages limit:1 %}
```

Comment:
What about replacing `for event` with `for page`?

Reply:
good idea, will do